project/README.rst

```diff
@@ -64,4 +64,4 @@ Follow the format to add your project in lexicographical order:
 
 1. `github_account_name <https://github.com/github_account_name>`__ for
```

Comment:
Please add an entry to link to your project proposal in `proposal/README.rst`.

Reply:
Done!
But I found out that the project list hasn't follow lexicographical order, should I fix it?

Reply:
Don't worry about it.  I will reorder later on.

I asked you "leave a global comment when you are ready for the next review".  It seems that you didn't get it.  The comment here is an inline comment, not a global one.  But it's OK.  You have addressed all points.  You probably need more experience to know what I meant.